src/pipeline.ts

```diff
@@ -421,6 +461,7 @@ export class GitHubWorkflow extends PipelineBase {
           contents: github.JobPermission.READ,
           idToken: this.useGitHubActionRole ? github.JobPermission.WRITE : github.JobPermission.NONE,
         },
+        environment: this.stackEnvs[stack.stackArtifactId],
```

Comment:
@kaizencc, I just tried the new feature and found one thing.
If ppl use the normal AddStage method, the output deploy.yml will have `environment: null`. Wonder if we shall skip adding the environment key if this.stackEnvs[stack.stackArtifactId] is null.
Possibliy:
```
...(this.stackEnvs[stack.stackArtifactId] && {
          environment: this.stackEnvs[stack.stackArtifactId],
        }),
```

Reply:
Hey @julianiag, caught me at a good time lol. Does that create an environment for you when you deploy? Or is it simply ignored? I agree it's not great. Maybe I can change it up tomorrow :).

Reply:
Or, if you add that in as a PR I will approve it, but you'll have to run `yarn build` to update the snapshots.

Reply:
https://github.com/cdklabs/cdk-pipelines-github/pull/186